extensions/Worker.Extensions.Kafka/src/Properties/AssemblyInfo.cs

```diff
@@ -2,5 +2,4 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
-
-[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.Kafka", "3.6.0")]
+[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.Kafka", "3.7.0")]
```

Comment:
nit: the line break between the using statement and the assembly attribute is present in all other extensions. So suggest you keep it to be consistent.

Reply:
Done